src/Tribe/Event_Scheduler.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Class Events_Scheduler
+ *
+ * Uses cron to move old events to trash and/or permanently delete them.
+ */
+class Tribe__Events__Event_Scheduler {
+
+	const DEL_CRON_HOOK = 'tribe-del-event-cron';
+	const TRASH_CRON_HOOK = 'tribe-trash-event-cron';
```

Comment:
To the best of my knowledge, class constants like this are harder to deprecate than just standard class properties, so [we try to avoid them](https://github.com/moderntribe/tribe-common/pull/331#pullrequestreview-21550954).

So I'd recommend replacing these constants with static class properties (with docblocks).